cmake/dts.cmake

```
@@ -34,6 +34,7 @@ list(APPEND
   DTS_ROOT
   ${APPLICATION_SOURCE_DIR}
   ${BOARD_DIR}
+  ${shield_dts_roots}
```

If you accept the other suggestion, this would become:

```suggestion
  ${SHIELD_DIRS}
```

---

@tejlmand That's a good idea, thanks. I am blocked, however, by #30553.